### ISSUE TITLE
Report about SCPH-102, SCPH-5502 not working

### ISSUE THREAD

**Issue by kubik369** [brad-lin/FreePSXBoot#27]
Hi,

firstly, thank you for the great project!

Secondly, I can report that my chipped SCPH-102 PSOne works with the 4.5 payload flawlessly - only thing that is different is that the screen doesn't blink after the exploit runs. It takes approximately 10 seconds, then shows bleak yellow-ish colour and after another ~10 seconds unirom comes up. In addition, I was able to format the card back to stock with MC Annihilator v2 on my SCPH-50004 PS2, the card did not crash the console (I also wrote the exploit with this setup).

I did the previous to test to see that my methodology is not incorrect, as the exploit does not work on my stock SCPH-5502 PAL system. I have been trying to make it work all of yesterday and even after a night with the console off and disconnected. I wasn't able to make it trigger at all, not even once.

I have a few questions:
- Is it possible that I have some uncommon BIOS revision on my SCPH-5502? I tried like 30 times and I didn't get the feeling that the exploit would be that unreliable.
- Is the only blocker for the 100% reliable exploit that you need testers? If it is so, I'm willing to test whatever you throw out there.

Thank you for any useful pointers :)

// EDIT: Also I forgot to mention, on my SCPH-102, when I select the memory card icon, it fades to black and stays like that, on the SCPH-5502, the memory card square icon fades to black and then reappears, afterwards the console freezes.

// EDIT2: Still no dice on the SCPH-5502. I have tried the 5501/5503/etc exploit version and it just goes into the memory card manager.

**Comment by whitezombie2000** [comment]
Hello, Can you open your SCPH-5502 model and look at the board revision? I'm curious if it's 1-664-537-52. You need to remove the six case black screws, disconnect the power supply cable, controller ribbon cable, cd power cable, cd ribbon cable, lift out the drive, remove two screw on the top of the metal shield, two screws on the bottom of the shield, gently lift the metal shield out, gently lift the thin copper like piece up, and you should be able to the number. I'm in the process of documenting my board revisions as I have several 5501 models. I have two NTSC SCPH-5501 models with a 1-664-537-52 board that won't boot this exploit.

**Comment by kubik369** [comment]
Sadly no, it's a PU-18 1-664-537-62. Could you point me to how to get the bios checksum? I would like to check it, but I'm afraid I won't be able to desolder the eeprom if that's required.

**Comment by whitezombie2000** [comment]
> Sadly no, it's a PU-18 1-664-537-62. Could you point me to how to get the bios checksum? I would like to check it, but I'm afraid I won't be able to desolder the eeprom if that's required.

Interesting...I have a NTSC SCPH-5501 PU-18 1-664-537-62 Sept 1997 and it boot the exploit.

**Comment by kubik369** [comment]
Thanks for the pointer. I will try to find some time tomorrow to burn it and try to disc-swap it in :)

**Comment by kubik369** [comment]
Well, the last blank CD-R that I had turned out to be bust so I will need to buy some more. I should be able to get some on Tuesday.

**Comment by ramapcsx2** [comment]
It is unlikely that we find a difference these days. Much more likely that you had a normally working exploit, but it's unreliable, and any further tests didn't have the RAM zeroe'd out yet. (Unless this is not a requirement anymore?)

**Comment by kubik369** [comment]
It is certainly possible, but I find it a little unlikely. I have been trying for the last 3 days at this point, each time leaving the console off for at least 10 minutes and longer (+ two nights unplugged without electricity and discharged). People seem to be reporting much better success rates even with the unreliable exploits, so me being that extremely unlucky seems a little weird.

As an aside, I have noticed, that my SCPH-5502 has the more modern grid menu (similar to PSOne), but my SCPH-9002 has the "older", colourful menu with splatters of paint. It seems to be normal after I searched, but I still find it odd that they would go back to the older menu in a newer revision
